config/webpack.config.dev.js

```diff
@@ -48,7 +48,11 @@ module.exports = {
     rules: [
       // Disable require.ensure as it's not a standard language feature.
       { parser: { requireEnsure: false } },
-
+      {
+        test: /\.mjs$/,
```

Comment:
In earlier rendition of the `hylo-shared` package the module transpire of the library was in an `.mjs` file. It's strange but standard extension to delineate between rpm modules transpired between `mjs` and `cjs` (CommonJS), etc. It's not a bad thing to have in our config in case another module is added that relies on that convention. Where I ended-up with `hylo-shared` doesn't use this format so it could be removed. I think I'll leave it for now, but maybe make a comment with some of the above.